js/main.js

```diff
@@ -22,8 +22,81 @@ function saveEntry(event) {
   dataObject.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(dataObject);
-  image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  var addNew = createDom(dataObject);
+  $parent.prepend(addNew);
+  $viewEntries.className = 'view';
+  $newModal.className = 'container hidden new';
+  $noEntry.className = 'hidden';
```

Comment:
You should handle this with your `switchView` function once you change it to accept an argument of the name of the view to show